app/views/task_lists/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="TaskLists index">
+  <h2>Task Lists</h2>
+
+  <%= link_to("Add new task list", new_task_list_path, :class => 'action new') %>
+
+  <table class="listing">
+    <caption>Task list</caption>
+    <tr class="header">
+    </tr>
+    <% @task_lists.each do |list| %>
+      <tr>
+        <td><%= list.name %></td>
+        <td class="actions">
+          <%= link_to("Show", task_list_path(list.id), :class => 'action show') %>
+          <%= link_to("Edit", edit_task_list_path(list), :class => 'action edit') %>
+          <%= link_to("Delete", task_list_path(list.id), form: { data: { turbo_confirm: "Are you sure you want to delete #{list.name}?" }}, :method => :delete) %>
```

Comment:
This works:
`<%= button_to "Delete", task_list_path(list.id), method: :delete %>`

But the confirmation still doesn't work, due to some fishy behavior in Rails turbo, which I can't fix right now.
So let's skip the confirm stuff ...